src/BalanceTx.purs

```diff
@@ -262,8 +266,9 @@ balanceTxM (UnbalancedTx { transaction: unbalancedTx, utxoIndex }) = do
         Just utxos, Just utxoIndex'' -> do
           let
             -- Combines utxos at the user address and those from any scripts
-            -- involved with the contract in the unbalanced transaction. Don't
-            -- add collateral to this for now.
+            -- involved with the contract in the unbalanced transaction.
+            -- For submitting a tx via Nami, this shouldn't include the collateral.
+            -- This is vital for balancing with Nami tx submission.
```

Comment:
If this is vital how about moving it into the function's docstring, that'd help with visibility.

Reply:
Good shout.